lib/active_model/serializer/logging.rb

```diff
@@ -0,0 +1,32 @@
+module ActiveModel
+  class Serializer
+    module Logging
+      extend ActiveSupport::Concern
+
+      included do
+        cattr_accessor(:logger) do
+          ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT))
+        end
```

Comment:
I prefer keep the `ActiveModel::Serializer.logger` since this is the way we define configs today like the adapter. More info about my rationale here https://github.com/maurogeorge/active_model_serializers/commit/d2c2378d55469a17fdef686c845207ae660a7bc2

In fact, what you think about using the config like: `ActiveModel::Serializer.config.logger` the same way we define the adapter we can define a custom log.